src/components/tabs/index.scss

```diff
@@ -0,0 +1,15 @@
+@import "src/components/style/base.scss";
+
+.tabs {
```

Comment:
Plz apply styles we to be look like on baseapp (wallet page). Not exact, but should be fealing that this is styled component